app/mainPage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { StyleSheet, View, ImageBackground, Text } from "react-native";
```

Comment:
```suggestion
import { StyleSheet, ImageBackground} from "react-native";
```

Use the themed text and view so we can do light/dark mode
`import { Text, View } from '@/components/Themed';`